Source/Add-ins/Profile.Common/DockableWindowMilSpaceProfileCalc.cs

```diff
@@ -957,6 +988,22 @@ public void SetPointInfo(PointTypesEnum pointType, string text)
             }
         }
 
+        public void SetReturnButtonEnable(ProfileSettingsPointButtonEnum pointType, bool enabled)
+        {
+            if(pointType == ProfileSettingsPointButtonEnum.PointsFist)
+            {
+                firstPointToolBar.Buttons["tlbbReturnPoint"].Enabled = enabled;
+            }
+            else if(pointType == ProfileSettingsPointButtonEnum.PointsSecond)
+            {
+                secondPointToolbar.Buttons["tlbbReturnToPoint"].Enabled = enabled;
+            }
+            else
+            {
+                basePointToolbar.Buttons["tlbbReturnCenterPoint"].Enabled = enabled;
```

Comment:
The tool buttons ca be accessed by their names  like tlbbReturnToPoint.Enabled = enabled